integration-tests/pom.xml

```diff
@@ -267,6 +267,7 @@
                 <module>micrometer-prometheus</module>
                 <module>opentelemetry</module>
                 <module>logging-json</module>
+                <module>jaxb</module>
```

Comment:
You need to add this new module to an existing slot in https://github.com/quarkusio/quarkus/blob/main/.github/native-tests.json